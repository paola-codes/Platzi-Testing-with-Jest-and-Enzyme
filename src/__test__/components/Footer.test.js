import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('Render of Footer component', () => {
    expect(footer.length).toEqual(1);
  });
  test('Render of Footer title', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });
});

describe('Footer Snapshot', () => {
  test('Verify UI of Footer component', () => {
    const component = renderer.create(<Footer />);
    const footer = component.toJSON();
    expect(footer).toMatchSnapshot();
  });
});

