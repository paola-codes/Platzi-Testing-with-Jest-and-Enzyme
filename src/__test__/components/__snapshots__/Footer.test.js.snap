// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Verify UI of Footer component 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;

exports[`Footer Snapshot2 Verify UI of Footer component 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
